lib/jekyll/tags/include.rb

```diff
@@ -252,21 +252,21 @@ def tag_includes_dirs(context)
         Array(page_path(context)).freeze
       end
 
+      private
+
```

Comment:
You should not change an existing public method into a private method unless you plan to bump the major version

Reply:
You may be right, although this method is only called within this class. There are also no separate tests or documentation for this method.